crates/bench/benches/subscription.rs

```diff
@@ -174,6 +198,24 @@ fn eval(c: &mut Criterion) {
             black_box(out);
         })
     });
+
+    // To profile this benchmark for 30s
+    // samply record -r 10000000 cargo bench --bench=subscription --profile=profiling -- query-indexes-many --exact --profile-time=30
+    // Iterate 1M rows.
+    bench_eval(
+        c,
+        "query-indexes-many",
+        "select * from location WHERE x = 0 AND z = 10000 AND dimension = 0",
+    );
+
+    // To profile this benchmark for 30s
+    // samply record -r 10000000 cargo bench --bench=subscription --profile=profiling -- query-indexes-multi --exact --profile-time=30
+    // Iterate 1M rows.
+    bench_eval(
+        c,
+        "query-indexes-multi",
+        "select * from location_multi WHERE x = 0 AND z = 10000 AND dimension = 0",
```

Comment:
We don't need both of these benchmarks if we only have the one location table.

Reply:
Done